src/Glpi/Controller/GenericAjaxCrudController.php

```diff
@@ -359,6 +342,10 @@ final protected function check(int $id, int $right, array $input): void
      */
     final protected function isClassAllowed(string $class): bool
     {
-        return in_array($class, CommonAjaxControllerAllowedClassesList::getClasses());
+        $allowed_classes = [
+            Form::class,
+        ];
+
+        return in_array($class, $allowed_classes);
```

Comment:
If the rights checks are correct, is there any reason to not enable it for all itemtypes ? 
We said we didn't want this allowlist to stay in the final version.